spec/checkPullRequestJobSpec.js

```diff
@@ -27,6 +27,9 @@ const checkPullRequestTemplateModule =
   require('../lib/checkPullRequestTemplate');
 const newCodeOwnerModule = require('../lib/checkForNewCodeowner');
 const scheduler = require('../lib/scheduler');
+const checkCronJobModule = require('../lib/checkNewCronJobs');
+const constants = require('../constants');
+const WIKI_LINK = constants.getWikiLink();
```

Comment:
Why do we need to redeclare this here?

Reply:
Link is also used in the tests for checkPullRequestJob ,so we have to call it from constants.js file

Reply:
I still don't get what you mean by this.

Reply:
Updated some chages PTAL